lib/tasks/argo.rake

```diff
@@ -18,6 +18,7 @@ end
 task :default => [:ci]
 
 task :ci do
+  raise 'NEVER RUN CI ON PROD' if ENV['RAILS_ENV'] =~ /production/i
   ENV['RAILS_ENV'] = 'test'
```

Comment:
Huh?  LN21 prevents it from running on prod.  Where else is it prevented?  Where else could it be prevented?